target/consensource-ui
# Convert components, common, and different persona views

## Proposed change/fix

- Convert components (forms, modals, testing banner)
- Convert common views (auth, profile, org)
- Convert persona views (main page, search, list, single views)
- Update Mithril (`m.route.link` -> `m.route.Link` component, remove `m.withAttrs`)
- Associated updates to services that were missed before to export correctly
- Add custom types in services

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test
Check intellisense, run `npm run lint` to ensure no errors on any `.ts` files.
`npm run build` to check for errors.
